.nvmrc

```diff
@@ -1 +1 @@
-v14.17.4
+v18
```

Comment:
Are you sure about that? 🙀 

I had lots of issue with Node 18 so I'm a bit traumatized, but if you think it's ok for our exercise, then let's go.

Reply:
They were due to changes in the module system, but I did not dig that much to provide more context.